media-api/app/controllers/MediaApi.scala

```diff
@@ -478,7 +478,7 @@ class MediaApi(
       // TODO: respondErrorCollection?
       errors => Future.successful(respondError(UnprocessableEntity, InvalidUriParams.errorKey,
         // Annoyingly `NonEmptyList` and `IList` don't have `mkString`
-        errors.map(_.message).list.reduce(_+ ", " +_), List(searchLink))
+        errors.map(_.message).list.toList.mkString(", ")) //.reduce(_+ ", " +_), List(searchLink))
```

Comment:
if I'm reading this right, can we delete the comment on ln.480 and remove the trailing comment on ln.481?

Reply:
ah nvm this was done in the subsequent commit 👍